dashboard/src/app/translation.initializer.ts

```diff
@@ -1,23 +1,45 @@
-import { catchError } from 'rxjs/operators';
-import { EMPTY } from 'rxjs';
 import { TranslateService } from '@ngx-translate/core';
+import { EMPTY } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
 import { EnplugService } from './services/enplug.service';
 
+const GET_USER_TIMEOUT = 2000;
+
 export function translationInitializer(enplug: EnplugService, translate: TranslateService) {
-  return () => new Promise<void>((resolve) => {
-    enplug.account.getUser().then(({ data: { locale } }) => {
-      if (locale && typeof locale === 'string' && locale.substr(0, 2) !== 'en') {
-        translate.getTranslation(locale).pipe(catchError(() => EMPTY))
-          .subscribe(
+  return () => new Promise<void>((resolve, reject) => {
+    let complete = false;
+
+    // Timeout to listen to getUser. If called from outside Enplug, will time out.
+    // Remove this for production code - it's here only for the tests
```

Comment:
We need to inform about this in a stronger way than a comment. I think that it might get omitted by many and it'd be left here.
Maybe a console warning?
Or being commented by default and then uncommenting it for testing